src/components/3d_editor/ThreeDEditorFullscreen.jsx

```diff
@@ -18,7 +18,7 @@ export class ThreeDEditorFullscreen extends mix(ThreeDEditor).with(FullscreenCom
                 leftIcon: <FullscreenIcon />,
                 onClick: () => {
                     clsInstance.setState({ viewerTriggerResize: true });
-                    clsInstance.goFullscreen();
+                    clsInstance.toggleFullscreen();
```

Comment:
why did package-lock change??

Reply:
You're right. This was an accident when I git-committed in the wrong terminal window.

Reply:
But it made the tests pass since the overnight change in MUI package. I'll merge dev to fix that.